app/javascript/json/supportedLanguages.json

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
I think this is a great idea, but I feel like it's more fitting in `app/config/` personally. @holmesie thoughts?

Reply:
It's front end, otherwise we would have to bubble up a yaml or something. I think it's a good idea, but not necessary for this PR.

Reply:
Works for me.